### ISSUE TITLE
php console not working on mac.. 

### ISSUE THREAD

**Issue by carlos-andres** [jonseg/crud-admin-generator#19]
Good Day, i try to use this excellent tool, but on my mac with MAMP and php 5.4.30 i get the follow errors, i really appreciate any help.. or there is a web interface?

thanks
PHP CONFIGURATION
![screen shot 2014-09-27 at 2 04 07 pm](https://cloud.githubusercontent.com/assets/5489747/4431801/749f9a60-4679-11e4-9fc0-11b4b6bfd68d.png)

PHP ERRORS
![screen shot 2014-09-27 at 2 04 16 pm](https://cloud.githubusercontent.com/assets/5489747/4431803/806bd124-4679-11e4-9ae3-c52b60af62db.png)

**Comment by pontigol** [comment]
Hi there, I have the exact same issue but when I try your solution I get "No such file or directory" there as well. I copied the admingenerator folder to htdocs, just as you did. I was wondering if you did any special tweaks in the app.php file: I only changed user/password and then host to localhost and db_name to my_db. I tried a lot of filepaths for asset_path, including just '/resources', which did you use? Or any other other changes that are not mentioned in the installation guide?

**Comment by blag001** [comment]
This seem to be an DB access related error (your php command need to access the mysql DB).

The best is trying to put this simple test code in a `test.php` file on your root `/www`

    <?php
    try {
        $dbh = new PDO('mysql:host=localhost;dbname=test', 'root, ''); // check user / pwd
        foreach($dbh->query('SELECT 40+2') as $row) {
            print_r($row);
        }
    } catch (PDOException $e) {
        die("Error!: " . $e->getMessage());
    }

Try running it with `php test.php` if you have an error, then take a look on google for "php command line connect mysql mac" or something alike.

if you still can't make it run, add some data about your config.

Regard,
Blag

**Comment by pontigol** [comment]
Hi again,

Thanks for you response! Actually I managed to fix it late last night, I just haven't had time to post it here. I found this discussion on stackoverflow:

http://stackoverflow.com/questions/6259424/troubleshooting-no-such-file-or-directory-when-running-php-app-console-doctri

...where they tell you to put 127.0.0.1:8889 as host (8889 is the default port in MAMP) and that fixed it! Now everything works well, so if there's any MAMP user out there struggling, try using 127.0.0.1:8889 as port.

Thanks again for your fast reply, hopefully both our answers will be helpful for others with similar problems.

Best regards,
Pontus
